### PR TITLE
Adding AlmaLinux after installation guide

### PR DESCRIPTION
**Pull request by sboldyreva**

### CODE REVIEW

File: docs/documentation/after-installation-guide.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+      ```
+
+To install Pantheon for AlmaLinux 9:
+* Enable EPEL, PowerTools and Synergy repositories
+* Install Pantheon Desktop Environment:
+  ```
+  sudo dnf groupinstall "Pantheon Desktop"
+  ```
+
+To have an AlmaLinux system with GNOME-Mini or MATE desktop environments we recommend installing AlmaLinux OS from the corresponding Live Media image.
```
Comment (bennyvasquez) [inline docs/documentation/after-installation-guide.md:230]: Should we link to the live media page here?

**Review by bennyvasquez — COMMENTED** [review]
Other than this one suggestion, it looks great!

File: docs/documentation/after-installation-guide.md (hunk 1 above)
Comment (sboldyreva) [reply docs/documentation/after-installation-guide.md]: @bennyvasquez linked and fixed a bit other links

**Comment by bennyvasquez** [comment]
Thanks, @sboldyreva! The .md's you added will break when it gets deployed, though, because they aren't part of the real site. For example: https://wiki.almalinux.org/repos instead of https://wiki.almalinux.org/repos.md

**Comment by sboldyreva** [comment]
@bennyvasquez thanks for noticing this, it should be fine now

**Review by bennyvasquez — APPROVED** [review]
Looks good!

**Pull request merged**